cli/src/plugins/javadoc/index.ts

```diff
@@ -936,27 +920,95 @@ const makeElementDetailBody: MakeBodyFunction = (args) => {
     .flat(1);
 };
 
+function uniqueify(s: string[]): string[] {
+  return [
+    ...new Set<string>(
+      s.filter((str) => str !== undefined && str !== null && str.trim() !== "")
+    ),
+  ];
+}
+
+function getParamNames(parameter: AnyType): string[] {
+  const paramNames: string[] = [
+    parameter.qualifiedTypeName,
+    parameter.simpleTypeName,
+    parameter.typeName,
+    processParam(parameter),
+  ];
+  return uniqueify(paramNames);
+}
+
+function getMethodNames(method: MethodDoc) {
+  return uniqueify([
+    method.qualifiedName,
+    method.name,
+    method.containingClass?.typeName + "." + method.name,
+  ]);
+}
+
+function getParamNameCombinations(
```

Comment:
What is this actually trying to do? Comment would be helpful. What is meant by "param combinations"?